src/main/java/testapp/core/UserAccount.java

```diff
@@ -0,0 +1,56 @@
+package testapp.core;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+
+import java.io.Serializable;
+
+public class UserAccount implements Serializable {
+    private Key userKey;
```

Comment:
why did you choose, to store the entire Key object instead of just the user's id value as long?